recipes-kernel/linux/files/bcm2712-raspberrypi5-domd.dts

```diff
@@ -63,6 +63,59 @@
 				      0x00 0x9e 0x04 0x00 0x9f 0x04 0x00 0xa0 0x04>;
 		};
 
+		mip1: msi-controller@131000 {
```

Comment:
I do not think we should take this node. In my opinion should be dropped.

Reply:
Dropped

Reply:
I am curious. Is this about MSI controller node? It is still present as far as I can see

Reply:
Yes. I forget to push changes